src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.ts

```diff
@@ -38,6 +38,14 @@ export class DatasetDetailsPanelComponent implements OnInit {
     return this.dataset.type === DatasetType.Filesystem && !isIocageMounted(this.dataset);
   }
 
+  get isAllowCapacity(): boolean {
```

Comment:
`isCapacityAllowed` or `hasCapacityCard`

Reply:
Will do that.

Reply:
Fixed